superx/routes/home.py

```diff
@@ -13,8 +13,7 @@ def home():
     returns landing page of application
     """
 
-    city_list = db.session.query(Branch.city).order_by(Branch.city).distinct().all()
-    return render_template('home.html', city_list=city_list)
+    return render_template('home.html')
```

Comment:
in #71  the render_template call added the city_list parameter, which is used on line 21 of home.html.
You might want to keep it.

Reply:
yah, realized that afterwards that i need to give them the option if not signed in